nexus/tests/integration_tests/external_ips.rs

```diff
@@ -1024,3 +1079,7 @@ async fn floating_ip_detach(
     .parsed_body()
     .unwrap()
 }
+
+fn assert_floating_ips_eq(first: &FloatingIp, second: &FloatingIp) {
+    assert_eq!(first.identity, second.identity);
+}
```

Comment:
Nitpick, but I think this could be inlined. It's not doing enough — I expected it to compare all fields or something.